src/docs/product/issues/issue-details/performance-issues/consecutive-db-queries/index.mdx

```diff
@@ -22,6 +22,7 @@ The detector for this performance issue looks for a set of sequential, non-overl
 Once these spans are found, the following must also hold true:
 
 - Maximum time saved from parallelization must exceed a threshold
+- The ratio between the maximum time saved and the duration of the sequential spans must exceed a threshold
```

Comment:
are these thresholds documented anywhere? might make sense to put them here for clarity

Reply:
The actual values themselves?

Reply:
If your referring to the values, they intentionally weren't added here for EA, as the thresholds could be tweaked. They will be added for GA

Reply:
yeah the value themselves. Ah ok, gotcha 👍